### DISCUSSION TITLE
Seeking Help: GPU Selection Issue

### DISCUSSION THREAD

**Discussion by Dead-Slash** [prasanthrangan/hyprdots#647] [Q&A]
Hello everyone,

I hope this message finds you well. As I continue to explore and adapt to Linux, I've encountered a challenge related to GPU selection in my system. Despite having an Nvidia GPU, the system consistently defaults to the integrated graphics. I'm reaching out to seek guidance on how to properly select and prioritize the Nvidia GPU for my tasks.
I've attached two screenshots for reference, illustrating the current configuration. It would be immensely helpful if anyone could provide insights or step-by-step instructions on how to ensure that the system utilizes the Nvidia GPU instead of the integrated graphics.
Thank you in advance for your support!

**Comment by ghost** [comment]
Okay, i'd appreciate it if you stopped completely copy pasting AI responses for help here, you didn't even include 2 screenshots. If you have problems with integrated graphics, i don't know exactly how because i don't have integrated graphics to test, but i'd look for more support online. I think you might even be able to do something in your BIOS.

**Reply by Dead-Slash** [reply]
Hello again i'm sorry about the use of AI but i'm from tunisia and my english is not that good so i pass through AI to be sure that i will make no fault and be understood so i ask AI to write what i have in mind in english that's all.
thank you
![231213_16h11m08s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/144330579/a723e65c-6d46-46ef-a20a-33164786ad50)

**Reply by ghost** [reply]
Ah okay, no worries! I'd just recommend you try to make the response less 100% copy+paste, you know?

Once again yeah your integrated gpu likely can be disabled with the BIOS. Also see if you can install nvidia drivers manually and if it just starts working. Maybe [this](https://wiki.archlinux.org/title/NVIDIA_Optimus) can be of use to you as well.

**Reply by Dead-Slash** [reply]
Thank you so much bro i will give it a try
